src/libs/ReportUtils.ts

```diff
@@ -1859,6 +1859,9 @@ function getPersonalDetailsForAccountID(accountID: number): Partial<PersonalDeta
 }
 
 const hiddenTranslation = Localize.translateLocal('common.hidden');
+
+const phoneNumberCache: Record<string, string> = {};
```

Comment:
Are there any other functions we could use this cache in this Utils file? Lets see if we can use it there in next PR

Reply:
I think the way forward for this would be to reuse the global cache proposed by Kacper in the other PR so we can swap every usage with this cached one in one go.

Reply:
This appears to be the only place in this file where the `formatPhoneNumber` is used